src/main/java/org/mondemand/Context.java

```diff
@@ -86,9 +86,9 @@ public boolean equals(Object obj) {
     return true;
   }
 
-  public void addContext(Client client)
```

Comment:
@nimaco This code was added in the previous version and had never been used. Do we still want a major version bump?